### ISSUE TITLE
User defined "overclocking" of cloudBit input stream rate

### ISSUE THREAD

**Issue by stresslimit** [littlebits/cloud-platform#3]
By default, cloudBits stream their input voltage at a rate of 750ms. This is ok but not super-amazing for real-time responsiveness. max = 750, min = 200? 100? 50?

- [ ] add to cloud commands: add custom stream rate to data model
- [ ] add endpoint to HTTP API
- [ ] add check for custom stream rate upon connection

**Comment by sch** [comment]
Am I mistaken, or do we already have this [in the settings](https://github.com/littlebitselectronics/cloud-control/blob/f9f4d9afe3bde24cd06244c51c29477ab58c842d/lib/view-dashboard/view-device-settings.js#L40-L56)? It's behind a feature flag — maybe I could add some more information about it in the cloud-control repo?

**Comment by jasonkuhrt** [comment]
Correct, behind a feature flag. This issue is quite close to being done. We just have to "ship" it.

**Comment by stresslimit** [comment]
Yeah we'd have to finalize the wording, and make sure our infra is ready. I suggest waiting til after `devicehub`2
